ReactNative/groceryList/src/components/ProductItem.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import {
+  Text,
+  TouchableOpacity
+} from 'react-native';
+import { CardLayout } from './common';
+
+class ProductItem extends Component {
+
+  onRowPress() {
+    const { product } = this.props;
+    const { uid, name, count, brand } = product;
+  }
+
+  render() {
+    return (
+      <CardLayout>
+        <TouchableOpacity onPress={this.onRowPress.bind(this)}>
+          <Text style={styles.textStyle}>
+            {this.props.product.name} - {this.props.product.brand} 
+          </Text>
+        </TouchableOpacity>
+      </CardLayout>
+    );
+  }
+}
+
+const styles = {
+  textStyle: {
+    fontSize: 18,
+    margin: 15
+  }
+};
```

Comment:
Let's try to import styles

Reply:
Sure, I just follow the tutorial. Maybe, we can do this kind of improvement in the future. So we will be able to create a new branch with all these improvements. What do you think?